cmsplugin_carousel_ai/models.py

```diff
@@ -13,6 +25,12 @@ class Carousel(CMSPlugin):
         verbose_name=_("name"),
         max_length=160,
     )
+    template = models.CharField(
+        verbose_name=_('Template'),
+        choices=get_templates(),
```

Comment:
As pointed in another comment, this isn't a good way to define the choices in a reusable application.